src/main/java/com/kaziamyr/onlinebookstore/service/ShoppingCartService.java

```diff
@@ -0,0 +1,16 @@
+package com.kaziamyr.onlinebookstore.service;
+
+import com.kaziamyr.onlinebookstore.dto.cartitem.CartItemDto;
+import com.kaziamyr.onlinebookstore.dto.cartitem.CreateCartItemRequestDto;
+import com.kaziamyr.onlinebookstore.dto.cartitem.PutCartItemRequestDto;
+import com.kaziamyr.onlinebookstore.dto.shoppingcart.ShoppingCartDto;
+
+public interface ShoppingCartService {
+    ShoppingCartDto getByUser();
```

Comment:
If I wrote comment 'It is clear that you working with shopping cart in this class' it means that all other methods should be changed as well, If it is not clear I can specify it in further reviews